docs/spelling_wordlist.txt

```diff
@@ -1297,6 +1297,7 @@ uid
 umask
 un
 unarchived
+uncommittedblobs
```

Comment:
```suggestion
```

Reply:
The test won't pass if this is removed

Reply:
This will work if you write each occurrence of this word as a code literal:
````
A ``uncommiitedblobs`` is an awesome thing
````
````
A uncommiitedblobs is an awesome thing
````
The first example causes no problems. The second will be raise warning by spellcheck.